plugins/modules/zos_copy.py

```diff
@@ -1490,21 +1520,12 @@ def copy_to_member(
         if self.is_binary:
             opts["options"] = "-B"
 
+        if self.is_executable:
+            opts["options"] = "-X"
```

Comment:
Noted as comment, Marcel and I discussed about the use of "I" option. This will keep aliases when copying, currently version 1.2.3 has some bugs with it, 1.2.4 appears to have fixed some of it.

Reply:
Agree I will add and run the pipeline again.